cloudinit/log/log_util.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import os
+import sys
+import time
+
+LOG = logging.getLogger(__name__)
+
+
+def logexc(
+    log, msg, *args, log_level: int = logging.WARNING, exc_info=True
+) -> None:
+    log.log(log_level, msg, *args)
+    log.debug(msg, exc_info=exc_info, *args)
+
+
+class log_time:
```

Comment:
> name bikeshed: Timed? Past tense because https://peps.python.org/pep-0343/#examples 

I like that a lot. Thanks for the suggestion.

> and no mention of log because it won't necessarily always log...which actually makes me wonder if this is the right place for this class.

+1 Maybe we do this instead?
```python
from cloudinit.performance import Timed
```